tq/verify.go

```diff
@@ -39,6 +39,7 @@ func verifyUpload(c *lfsapi.Client, remote string, t *Transfer) error {
 		req.Header.Set(key, value)
 	}
 	req.Header.Set("Content-Type", "application/vnd.git-lfs+json")
+	req.Header.Set("Accept", "application/vnd.git-lfs+json")
```

Comment:
No, this will break compatibility with lfs-test-server and current Gitea. See git-lfs/lfs-test-server#85. Change the order: first set your headers and **then** overwrite them with headers from `action.Header`.

Reply:
Okay, fixed and squashed.